### ISSUE TITLE
Considering past rejected attempts

### ISSUE THREAD

**Issue by oxinabox** [stevengj/subsuper-proposal#8]
Looking at https://www.quora.com/Why-is-there-no-character-for-superscript-q-in-Unicode

It mentions some past attempts:

**superscript q**

 - http://www.unicode.org/L2/L2010/10230-modifier-q.pdf

These comments were found:
 - http://www.unicode.org/L2/L2010/10315-comment.pdf
 - http://www.unicode.org/L2/L2010/10316-cmts.pdf

Also 
**All subscripts and superscript letters:**
 - http://www.unicode.org/L2/L2011/11208-n4068.pdf

I think there is probably a way to use document numbers to look up more about the rejections.

**Comment by stevengj** [comment]
I think I already commented on these previous rejections in the draft; did I miss something?

**Comment by oxinabox** [comment]
Sorry it looks like I never completed my thoughts before posting the issue.

You give some solid reasons for the difference for sure.
But I think it would be good to point-by-point quote and refute the rejection reasons.
Either  in the document itself, or in an issue, to just make sure our coverage is *tight*.

This means finding the formal rejection notices.
I'll give that a shot shortly.

**Comment by stevengj** [comment]
Yes, if you can find the formal rejection notices that would be extremely helpful!  I never managed to locate those.

**Comment by oxinabox** [comment]
[the minutes](http://www.unicode.org/L2/L2010/10221.htm#124-A146) where the proposal for a q was rejected are terse to say the least
I suggest that it is worth contacting Karl Pentzlin.
Not only to get the exact rejection reason.
But because he is someone who has made many proposal's to the Unicode Technical Committee.
His feedback would be valuable.

It looks like:  	LATIN LETTER SMALL CAPITAL Q
Is already incoming
AllocationL A7AF 	
Count: 1
Name: LATIN LETTER SMALL CAPITAL Q 	
UTC Status 2016-Jan-25 Accepted 	
ISO StageL 2016-Jun-07 Stage 5

http://www.unicode.org/alloc/Pipeline.html


Re: http://unicode.org/L2/L2011/11208-n4068.pdf
I have found some more feedback:
from German NB http://unicode.org/L2/L2011/11227-n4085.pdf
But I can't find it in the minutes:
It was a agendaed as C.2: http://www.unicode.org/L2/L2011/11260.htm
But never discussed AFAICT: http://www.unicode.org/consortium/utc-minutes/UTC-128-201108.html

**Comment by stevengj** [comment]
Because the super/subscript Latin characters were added at different times for different reasons, they don't necessarily go well together in any given font.  My current thinking is to suggest that the complete set be re-added as "mathematical superscript a", "mathematical superscript b", etcetera.

**Comment by stevengj** [comment]
I sent Pentzlin an email.

**Comment by m93a** [comment]
@stevengj Did you recieve a response? Did you learn something valuable you'd like to share?

**Comment by stevengj** [comment]
I still haven't gotten around to finishing this proposal up and submitting it…

**Comment by thchr** [comment]
Just happened upon this semi-recent [related proposal (L2/18-206)](http://www.unicode.org/L2/L2018/18206-super-sub-scripts.pdf) which seems to have been [rejected in 2018](https://www.unicode.org/L2/L2018/18241-script-ad-hoc.pdf) (section 1.b, p. 2). 
The L2/18-206 proposal doesn't seem particularly thorough but I figured it might be worthwhile to point it out here - and particularly to highlight the consortium's comments: 

>**Document:** [L2/18-206](http://www.unicode.org/L2/L2018/18206-super-sub-scripts.pdf) 1120 more superscripts, subscripts and small capitals – Grochowski
>
>**Comments:** We reviewed this document, which requests adding 1,120 superscripts, subscripts, and
small capitals. In our opinion, adding 1,120 such characters is not a good idea architecturally. A full
proposal with orthographic evidence for the specific characters could, however, be considered, if the
author provided such a document.
>
>**Recommendations:** We recommend the UTC note this document, relaying to the author that if
orthographic evidence were provided, the request could be re-considered (although a full proposal is
required).

I'm not completely clear on what they mean by "orthographic evidence", but I'm assuming it's coming back to the point raised elsewhere in this repo that the typesetting of sub/superscripts itself need to be more than a mere resizing.

PS. This strange guess-if-the-sub/superscript-exists is such a pain.

**Comment by stevengj** [comment]
I think "orthographic evidence" refers to evidence of the characters being in use and having semantic meaning distinct from formatting.

**Comment by thchr** [comment]
A separate attempt was made in 2021 with the [L2/21-043](https://www.unicode.org/L2/L2021/21043-subscript-mod-ltr.pdf) proposal, but also failed.
To my mind, it's a nice, well-structured proposal, arguing the need for subscript variants from a different angle (mainly phonetics usage).

The Unicode voting group unfortunately were also not swayed by this proposal, [opting to simply note the proposal but take no action](https://www.unicode.org/L2/L2021/21016-script-adhoc-rept.pdf):

> **Document:** [L2/21-043](http://www.unicode.org/L2/L2021/21043-subscript-mod-ltr.pdf) Unicode request for subscript modifier-letter support – Miller
> **Comments:** This proposal covers subscript modifiers in a variety of traditions. The proposed subscript modifier letters were never in official IPA usage.
>
> During discussion there was concern that some of the figures do not make a clear case for the proposed subscripts to be carried in plain text. Rather, the evidence shows mixed orthographic conventions.
> Figures 7 and 8, for example, demonstrate a subscripting convention for segmental or morphological analysis that is already representable as rich text. A more convincing scenario would be evidence of two authors communicating a transcription via print using the same subscript convention.
>
> **Recommendation:** We recommend the UTC make the following disposition: 
> Notes this document (L2/21-043) but takes no further action.

Relative to the [rejection](https://www.unicode.org/L2/L2018/18241-script-ad-hoc.pdf) of the 2018 [L2/18-206](http://www.unicode.org/L2/L2018/18206-super-sub-scripts.pdf) proposal, the critique has now moved from a lack of "orthographic evidence" to "evidence shows mixed orthographic conventions" 🤷 .

**Comment by thchr** [comment]
Kirk Miller, author of [L2/21-043](https://www.unicode.org/L2/L2021/21043-subscript-mod-ltr.pdf) above, made another, more limited attempt in 2024 with [L2/24-219](https://www.unicode.org/L2/L2024/24219-subscript-w-y-z-gamma.pdf), requesting additions of w, y, z, and $\gamma$ subscripts. As far as I can tell, the proposal has not yet been assessed, however.

**Comment by oameye** [comment]
@thchr Really appreciate you logging these attempts here :)

**Comment by thchr** [comment]
The 2024 proposal to add w, y, z, and $\gamma$ subscripts by Kirk Miller in [L2-24-219](https://www.unicode.org/L2/L2024/24219-subscript-w-y-z-gamma.pdf) appears to have been accepted in late 2024, cf. https://www.unicode.org/L2/L2024/24228-script-wg-report.pdf, with the attached comment that:

> This document proposes 4 code points for Latin subscript letters w, y, z and gamma used in several phonetic transcription systems. The evidence was found satisfactory.

The code points have also been assigned (I am guessing for Unicode 18?), cf. p. 11 of https://www.unicode.org/L2/L2024/24268-n5291-post-17-chart.pdf.
